api/v1beta1/disruption_webhook.go

```diff
@@ -39,8 +40,13 @@ var namespaceThreshold float64
 var clusterThreshold float64
 var handlerEnabled bool
 var defaultDuration time.Duration
+var ddmarkLibPath = "chaos-api"
```

Comment:
Any chance of making this a const and defining it in only one place across the codebase?

Reply:
added to types, it is easier this way